Add download link banner to Thunderbird post-download donation page.

Sancus opened the pull request:
Hey,

We'd like to add a banner linking back to the Thunderbird site on the post-download donation page. The reason is because some people are confused by the redirect, or their download doesn't start, making them feel forced to donate or unable to download from the front page.

Here's what that looks like: ![image](https://user-images.githubusercontent.com/479671/44243367-9ce76c00-a183-11e8-9875-d07824c9dbf7.png)

I haven't done much with React before, so let me know if there's a better way to do any of this, I just chose what appeared to be the least invasive way.

kristinashu commented:
Looks good! My only recommendation would be to use the sans serif font Nunito (similar to the "Thank you for downloading Thunderbird" text on the left site) for the text and maybe to move the banner up so it's against the top of the browser.

Here's a quick mockup:
![thunder](https://user-images.githubusercontent.com/1068783/44365896-dec42b00-a47f-11e8-86e8-1939da125467.jpg)


For reference, we have one like this on the foundation site:
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/1068783/44365433-a5d78680-a47e-11e8-94d3-f6b2f979651f.png">

Sancus commented:
I've made the changes requested, fixed the code issues and changed the font and font weight, new screenshot:

![image](https://user-images.githubusercontent.com/479671/44369333-dfae8a00-a48a-11e8-9cbd-bf8c9671d2e9.png)

kristinashu commented:
Design looks good to me!
